### PR TITLE
add new task, Matrix11

### PR DESCRIPTION
**Pull request by marina-nazarenko**

### CODE REVIEW

File: src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package javaTutor.algorithmization.twoDimensionalArrays;
+
+import javaTutor.utils.ArrayUtils;
+
+public class Matrix11 {
+
+    public static void main(String[] args) {
+
```
Comment (denis-tumel) [inline src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java:8]: redurand line

File: src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        int sizeX = 20;
+        int sizeY = 10;
+        int[][] matrix = matrix11.generationMatrix(sizeX, sizeY);
+        String array = matrix11.findLinesWithNumberFive(matrix);
+        System.out.println("Номера строк, в которых цифра 5 встречается 3 и более раз: ");
+        if (array.equals("")) {
+            System.out.println("Нет таких строк!");
+        } else {
+            System.out.println(array);
+        }
```
Comment (denis-tumel) [inline src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java:19]: 14 - 19 move to separate private method with name `outputResult` or something else

File: src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java (hunk 1 above)
Comment (marina-nazarenko) [reply src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java:8]: fixed

File: src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java (hunk 2 above)
Comment (marina-nazarenko) [reply src/javaTutor/algorithmization/twoDimensionalArrays/Matrix11.java:19]: fixed

**Pull request merged**